evm/src/cpu/kernel/asm/bignum/util.asm

```diff
@@ -1,13 +1,15 @@
-%macro memcpy_kernel_general
+%macro memcpy_current_general
     // stack: dst, src, len
-    %stack (dst, src, len) -> (0, @SEGMENT_KERNEL_GENERAL, dst, 0, @SEGMENT_KERNEL_GENERAL, src, len, %%after)
+    GET_CONTEXT
+    %stack (context, dst, src, len) -> (context, @SEGMENT_KERNEL_GENERAL, dst, 0, @SEGMENT_KERNEL_GENERAL, src, len, %%after)
```

Comment:
Not entirely sure about how these macros are used, but shouldn't the `0` be a `context`?

Reply:
Ah good catch!